include/VSSPath.hpp

```diff
@@ -29,17 +29,23 @@ class VSSPath {
         std::string getVSSPath() const;
         std::string getVSSGen1Path() const;
         std::string getJSONPath() const;
+        std::string to_string() const;
         bool isGen1Origin() const;
         static VSSPath fromVSSGen2(std::string vss); //Expect Gen2 path
         static VSSPath fromVSSGen1(std::string vssgen1); //Expect Gen1 path
-        static VSSPath fromJSON(std::string json); //Expect json path
+        static VSSPath fromJSON(std::string json, bool gen1); //Expect json path
```

Comment:
Why this parameter? JSONPAth do not have "gen1/gen2" difference, they are JSONpath expressions, so I do not understand the meaning here.

Reply:
JSON has no difference for gen1 and gen2, therefore, external input needed, therefore the parameter here to construct a new VSSPath

Reply:
But there should no usecase for this ever? "gen1" and "gen2" is only ever relevant when constructing from an incoming JSON query, when do we ever construct a fresh VSSPath from a JSONPath where that is relevant?
At least there should be a version where it is not needed/set to default